index.html

```diff
@@ -1,15 +1,24 @@
 ---
-layout: index
+layout: default
 title: FOSS Family Recipes
 hero: /images/potatoes-and-pot.jpg
 ---
-<header>
-    <h1>{{ page.title }}</h1>
-    <p>Free and community-sourced recipes.</p>
-</header>
 
-<ul class="actions">
-    <li><a href="/categories" class="button big">Tags</a></li>
-    <li><a href="/recipes" class="button big">Browse All</a></li>
-    <li><a href="https://github.com/thenaterhood/fossrecipes" class="button big">Github</a></li>
-</ul>
+<!-- Banner -->
+<section id="banner">
+    <div class="content">
+        <header>
+            <h1>{{ page.title }} </h1>
+            <p>A free and fully responsive site template</p>
```

Comment:
Looks like something from the default template that should be removed or changed.